python/lsst/obs/base/script/ingestRaws.py

```diff
@@ -67,9 +67,5 @@ def ingestRaws(repo, output_run, config=None, config_file=None, directory=None,
             configOverrides.addValueOverride(name, value)
     configOverrides.applyTo(ingestConfig)
     ingester = TaskClass(config=ingestConfig, butler=butler)
-    files = [file] if file is not None else []
-    if directory is not None:
-        suffixes = (".fits", ".fits.gz", ".fits.fz")
-        files.extend(
-            [os.path.join(directory, f) for f in os.listdir(directory) if f.lower().endswith(suffixes)])
```

Comment:
That `f.lower()` is interesting, implies that you want to use a case insensitive regex.

Reply:
I don't think so? This is removed code that tests for 3 specific file endings, case insensitive. But it's been replaced by `findFiles` (rename TODO) based on your regex impl.

Reply:
I know that it's removed code. My point is that the replacement regex is not the same as the code that is being replaced, in terms of functionality. To be consistent we need to enable case insensitive regex. The old code matched .FITS and .fits but the new code won't find .FITS files. I assume that code path was added for a reason. Maybe it's not a problem but acknowledging that we have changed behavior here is still worthwhile (we won't be able to ingest a .FITS file without also fixing the formatter so maybe we fix this when someone complains that a .FITS file isn't found). @TallJimbo did you have a real use case for .FITS?

Reply:
I did not.

Reply:
Ok, let it go for now then and we can tweak the regex later if we need to.

Reply:
I see, thanks.